invenio_rdm_pure/source/rdm/database.py

```diff
@@ -59,17 +61,24 @@ def select_query(self, fields: str, table: str, filters={}):
             return False
         return self.cursor.fetchall()
 
-    def get_pure_admin_userid(self):
-        """Gets the userId of the Pure admin user."""
-        email = current_app.config.get("INVENIO_PURE_USER_EMAIL")
-        email = f"'{email}'"
-        response = self.select_query("id", "accounts_user", {"email": email})
-        if not response:
-            report = """
-            ERROR:
-            Pure admin user not found.
-            Make sure that the user is created and that it is stored in data_setup/rdmUser_pureEmail.txt
-            """
-            self.report.add(report)
-            return False
-        return response[0][0]
+    def get_pure_user_id(self):
+        """Gets the userId of the Pure user.
+
+        In case the user doesn't exist yet,
+        it is created with credentials defined in config.py.
+        """
+        datastore = current_app.extensions["security"].datastore
+        if datastore is not None:
```

Comment:
Add a short comment on ```datastore```. 
for example:

"""fetching user object with ```flask_security.datastore```."""

Question: wouldn't it be better to fetch with the class itself:

```
from flask_security.datastore import UserDatastore
invenio_pure_user = UserDatastore.get_user(invenio_pure_user_email)
```
and get rid of the datastore?
It's only better when writing tests for the module. and then we don't need to add the security to the extension.

Reply:
It sounds like a good idea, but I am not completely sure if that has the context of the module. 
I saw the user creation happening in this way [here](https://github.com/fair-data-austria/invenio-madmp/blob/74372ee794f81666f5e9cf08ef448c21b2e428be/tests/conftest.py#L139).

Additionally, as of now the implementation of get_user is missing:
```
def get_user(self, id_or_email):
        """Returns a user matching the specified ID or email address."""
        raise NotImplementedError
```